test/bats/configs/vault.yaml

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
Why not just use our helm chart instead of rolling your own?

Reply:
Good point, this was mostly due to keeping the same method as the tests these were based on. I've switched to installing via helm chart now.